jwst/stpipe/_cal_logs.py

```diff
@@ -29,6 +29,14 @@ def _scrub(msg):
     scrubbed : str
         The scrubbed string
     """
+    if "/" + _USER + "/" in msg:
```

Comment:
@braingram @tapastro The solution I went with here seems a bit unrefined, but on the other hand I couldn't find anything obviously wrong with it.  You're better than me at that sort of thing... thoughts?

Reply:
I love regex... can you help save me some time and explain what advantages this has over my proposed solution?

Reply:
Maybe speed (but that would be hard to test given the variability in input) but also that the regex allows dropping the splits, joins, and custom logic. The regex would be less likely to raise an exception on an unexpected input (whereas the custom logic might).